website/templates/project/contributors.mako

```diff
@@ -8,245 +8,363 @@
-                        </th>
-                        <th class="col-md-3">
-                            Bibliographic Contributor
-                            <i class="fa fa-question-circle visibility-info"
-                                    data-toggle="popover"
-                                    data-title="Bibliographic Contributor Information"
-                                    data-container="body"
-                                    data-placement="right"
-                                    data-html="true"
-                                ></i>
-                        </th>
-                        <th class="col-md-1">
-                        </th>
-                        </tr>
-                    </thead>
-                    <tbody data-bind="sortable: {
-                            template: 'contribTpl',
-                            data: contributors,
-                            as: 'contributor',
-                            isEnabled: canEdit,
-                            options: {
-                              containment: '#manageContributors'
-                            }
-                        }">
-                    </tbody>
-                  </table>
-
-                <div data-bind="if: adminContributors.length">
-                    <h4>
-                      Admins on Parent Projects
-                      <i class="fa fa-question-circle admin-info"
-                              data-content="These users are not contributors on
-                              this component but can view and register it because they
-                                are administrators on a parent project."
-                              data-toggle="popover"
-                              data-title="Admins on Parent Projects"
-                              data-container="body"
-                              data-placement="right"
-                              data-html="true"
-                          ></i>
-                    </h4>
-                    <table class="table">
-                        <thead>
-                            <tr>
-                                <th class="col-md-6"></th>
-                                <th class="col-md-2"></th>
-                                <th class="col-md-3"></th>
-                                <th class="col-md-1"></th>
-                            </tr>
-                        </thead>
-                        <tbody data-bind="template: {
-                                name: 'contribTpl',
-                                foreach: adminContributors,
-                                as: 'contributor'
-                            }">
-                        </tbody>
-                    </table>
-                </div>
-                ${buttonGroup()}
+<div class="col-md-3 col-xs-12">
+    <div class="filters">
+        <input type="text" class="form-control searchable" id="nameSearch" placeholder="Search by name"/>
```

Comment:
Was this placeholder text in the spec for this feature? It could be misleading; a user might think that this searches amongst all users, and that this input is for adding a contributor. Perhaps change to "Filter by name" to indicate that this is filtering amongst existing contributors.

This is minor, and it shouldn't block merge.